### PR TITLE
Use highmem node pool for copy_deduplicate_all

### PR DESCRIPTION
**Pull request by scholtzan**
Run `copy_deduplicate_all` on the highmem node pool

### CODE REVIEW

File: dags/copy_deduplicate.py [hunk 1]
```diff
@@ -33,13 +33,20 @@
...
         target_project_id="moz-fx-data-shared-prod",
         priority_weight=100,
         # Any table listed here under except_tables _must_ have a corresponding
         # copy_deduplicate job in another DAG.
-        except_tables=["telemetry_live.main_v4"])
+        except_tables=["telemetry_live.main_v4"],
+        kwargs={
+            "node_selectors": {"nodepool" : "highmem"},
+            "resources": resources
+        })
```
Comment (jklukas) [inline dags/copy_deduplicate.py:49]:
```suggestion
        node_selectors={"nodepool" : "highmem"},
        resources=resources)
```

The last argument of `bigquery_etl_copy_deduplicate` is `**kwargs` which captures all additional keyword arguments. I believe that passing literal `kwargs` here would end end up as `kwargs["kwargs"]` in the destination function.

**Review by jklukas — CHANGES_REQUESTED** [review]

File: dags/copy_deduplicate.py (hunk 1 above)
Comment (scholtzan) [reply dags/copy_deduplicate.py]: Ah indeed

**Review by jklukas — APPROVED** [review]
LGTM. I'm concerned about why this job would be consuming substantial memory at all, so not sure whether this will make a difference. But I think it's worth trying, and we should definitely kick off a rerun during the day today to be sure this works.

**Comment by haroldwoo** [comment]
I did add another node to the default pool today. You may want to hold off on merging this to see if that helps tonight.

Looking at the other 2 nodes in the default pool, there are memory usage spikes, although they could be attributed to another GKE job.

**Comment by scholtzan** [comment]
Alright, let's wait and see what happens tonight. If we see the same issue then I'll merge this.

**Comment by scholtzan** [comment]
copy_deduplicate_all did fail again. I'll merge this and then rerun the task

**Pull request merged**